renovate.yaml

```diff
@@ -0,0 +1,10 @@
+# renovate: datasource=github-releases depName=mozilla/sccache
+SCCACHE: v0.7.6
```

Comment:
I think these names should match what we use elsewhere (eg `CUDA_VER`, `PYTHON_VER`, etc).

I think looking at the Dockfiles with `ARG YQ` and `ARG CODECOV` doesn't necessarily capture what they are for. But `YQ_VER` and `CODECOV_VER` do.